website/static/css/pages/home-page.css

```diff
@@ -35,3 +35,8 @@ h3, h4 {
     background-color: #52A452;
     border-color: #54AC54;
 }
+
+.btn-view-meetings {
```

Comment:
Realized this after merging, but this can have a more generic name which describes what the class does, e.g. `btn-success-high-contrast`.